blueocean-git-pipeline/pom.xml

```diff
@@ -37,7 +37,7 @@
       <groupId>org.eclipse.jgit</groupId>
       <artifactId>org.eclipse.jgit.ssh.jsch</artifactId>
       <!-- TODO this is hard to manage; if needed, should be in BOM -->
-      <version>6.5.0.202303070854-r</version>
+      <version>6.6.0.202305301015-r</version>
```

Comment:
Aligned with the version shipped in the version of the Git plugin shipped in BOM 2278.v47b_4508e256a (consumed below).